### PR TITLE
[ContainerRegistry] ACR orderby

### PR DESCRIPTION
**Pull request by seankane-msft**

### CODE REVIEW

File: sdk/containerregistry/azure-containerregistry/azure/containerregistry/aio/_async_container_repository_client.py [hunk 1]
```diff
@@ -194,6 +196,8 @@ async def set_tag_properties(self, tag: str, permissions: ContentPermissions, **
         :returns: None
         :raises: None
```
Comment (mccoyp) [inline sdk/containerregistry/azure-containerregistry/azure/containerregistry/aio/_async_container_repository_client.py:197]: Raises and/or return type seems to be missing here

File: sdk/containerregistry/azure-containerregistry/azure/containerregistry/aio/_async_container_repository_client.py [hunk 2]
```diff
@@ -178,13 +179,14 @@ async def set_manifest_properties(self, digest: str, permissions: ContentPermiss
...
```
Comment (mccoyp) [inline sdk/containerregistry/azure-containerregistry/azure/containerregistry/aio/_async_container_repository_client.py:180]: Raises and/or return type seems to be missing here (type hint as well)

File: sdk/containerregistry/azure-containerregistry/tests/test_container_repository_client_async.py [hunk 3]
```diff
@@ -168,3 +163,76 @@ async def test_set_manifest_properties_does_not_exist(self, containerregistry_ba
...
+
+
+    @acr_preparer()
+    async def test_list_registry_artifacts(self, containerregistry_baseurl):
+        client = self.create_repository_client(containerregistry_baseurl, self.repository)
+
+        count = 0
+        async for artifact in client.list_registry_artifacts():
+            assert artifact is not None
+            assert isinstance(artifact, RegistryArtifactProperties)
```
Comment (swathipil) [inline sdk/containerregistry/azure-containerregistry/tests/test_container_repository_client_async.py:203]: is there a reason for checking `not None` before checking `isinstance`? I would think `isinstance` assertion would encompass `not None`?

Comment (seankane-msft) [reply sdk/containerregistry/azure-containerregistry/tests/test_container_repository_client_async.py:203]: You could absolutely do this. Mostly a matter of habit.

**Review by mccoyp — APPROVED** [review]
My comments were addressed in another PR and the rest looks good!

**Pull request merged**